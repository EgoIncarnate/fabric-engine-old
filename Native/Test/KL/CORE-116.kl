struct Foo { Integer bar; } ;
struct Foo { Scalar bar; } ;

function entry()
{
}
