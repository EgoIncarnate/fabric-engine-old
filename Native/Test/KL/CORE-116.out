2:1: error: type 'Foo' already registered with a different definition
